redfish-core/src/error_messages.cpp

```diff
@@ -928,6 +928,40 @@ void propertyValueIncorrect(crow::Response& res, const std::string& arg1,
     addMessageToErrorJson(res.jsonValue, propertyValueIncorrect(arg1, arg2));
 }
 
+/**
+ * @internal
+ * @brief Formats PropertyValueResourceConflict message into JSON
+ *
+ * See header file for more information
+ * @endinternal
+ */
+nlohmann::json propertyValueResourceConflict(const std::string& arg1,
+                                             const std::string& arg2,
+                                             const std::string& arg3)
+{
+    return nlohmann::json{
+        {"@odata.type", "#Message.v1_1_1.Message"},
+        {"MessageId", "Base.1.10.0.PropertyValueResourceConflict"},
```

Comment:
Is this the only one that uses  1.10?
I see       {"MessageId", "Base.1.8.1.ResourceCreationConflict"}, ?

Reply:
There are few other properties that have used the respective major and minor versions of the message registry like this.
```
-bash-4.2$ git grep -in "MessageId" -- redfish-core/src/error_messages.cpp | grep -v 1.8.1
redfish-core/src/error_messages.cpp:587:        {"MessageId", "Base.1.11.0.ServiceDisabled"},
redfish-core/src/error_messages.cpp:944:        {"MessageId", "Base.1.10.0.PropertyValueResourceConflict"},
redfish-core/src/error_messages.cpp:1392:        {"MessageId", "Base.1.12.1.StrictAccountTypes"},
redfish-core/src/error_messages.cpp:2204:        {"MessageId", "OpenBMC.0.2.0.InvalidUpload"},
redfish-core/src/error_messages.cpp:2223:        {"MessageId", "Base.1.5.0.MutualExclusiveProperties"},
redfish-core/src/error_messages.cpp:2251:        {"MessageId", "Base.1.9.0.RestrictedRole"},
```

I think this will go away once we migrated to the upstream since its added based on the base header id - Please refer [here](https://github.com/openbmc/bmcweb/blob/master/redfish-core/src/error_messages.cpp#L129)